projects/vrpano.md

```diff
@@ -0,0 +1,95 @@
+## Project Name
+<!-- Add your project name here with format "Project Name"-->
+VR Pano
+
+## Category 
+<!--developer tooling, application, wallet, infrastructure, etc-->
+VR Pano Viewer, 3D Model Viewer, Filcoin's 3D Data Marketplace
+
+## Project Description
+<!--Describe your project in a few sentences. -->
+
+VRPano uses the distributed storage feature of block chain to realize cross-platform and multi-format content storage and distribution mechanism, and supports scene models, character models, VR movies, AR models and other materials. The cost is 30% lower than traditional object storage. Meanwhile, VR content can be traceable, copyright protection, digital asset trading and other services.
+
+
+## Project Status
+<!--brainstorming, fundraising, under development, beta, shipped, etc-->
+- [doing] Beta
+- [done] Demo
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
+![Vrpano Pre1](http://106.12.254.207:8006/ipns/bafzbeig6wke4thvbv3bbydhb6eflyt5w3iwxo4vb3lr5h7i5jwkzwcm22m/screenshot/vrshow1.png)
+![Vrpano Pre2](http://106.12.254.207:8006/ipns/bafzbeig6wke4thvbv3bbydhb6eflyt5w3iwxo4vb3lr5h7i5jwkzwcm22m/screenshot/vrshow2.png)
+![Vrpano Pre3](http://106.12.254.207:8006/ipns/bafzbeig6wke4thvbv3bbydhb6eflyt5w3iwxo4vb3lr5h7i5jwkzwcm22m/screenshot/vrshow3.png)
+
```

Comment:
Howdy! This address is not reachable, can you use an IPFS gateway address like ipfs.io or dweb.link or cloudflare?